dns/outputs.tf

```diff
@@ -8,6 +8,11 @@ output "provisionaccount_role" {
   description = "The IAM role that allows sufficient permissions to provision all AWS resources in the DNS account."
 }
 
+output "provisionpublishegressip_role" {
+  value       = aws_iam_role.provisionpublishegressip_role
+  description = "The IAM role that allows sufficient permissions to provision all resources related to the publish-egress-ip Lambda in the DNS account."
+}
+
```

Comment:
Please properly alphabetize this additional output.

Reply:
Excellent catch- rectified in https://github.com/cisagov/cool-accounts/pull/127/commits/d2de7cdf53f981720d62dcd11299d997c9a7d785.

Reply:
Your change is also nice, but I was actually referring to the fact that `primary_delegation_set` should come before `provisionaccount_role` and `provisionpublishegressip_role`.

Reply:
Haha, I totally missed that before, thanks. 😄  See https://github.com/cisagov/cool-accounts/pull/127/commits/69365712af3d7198735f4f6e3d916f20662dd485.